src/App/ComponentsDocumentation/components/Sheet/index.js

```diff
@@ -1,85 +1,87 @@
+                </div>
+                <p>Do not allow horizontal scrolling in a Sheet or layout that suggests horizontal scrolling.</p>
+            </div>
+        </div>
+        <div className="row">
+            <div className="col-12">
+                <div className="slab slab-plain slab-border-success">
+                    <h3>Do</h3>
+                    <div className="d-flex flex-column align-items-center">
+                        <img src={`${basename}img/documentation/sheet/do2.png`}alt="Sheet component in mobile view. Displaying filter example." className="img-fluid"/>
                     </div>
-                    <button className="btn btn-primary" type="submit" onClick={e => e.preventDefault()}>Submit</button>
-                </form>
-            </SheetComponent>{"\n"}
-            <button className="btn btn-primary" type="button" data-sheet-open="demo-sheet">Open sheet</button>
-        </ComponentPreview>
+                </div>
+                <p>A Sheet can be used as a container of a <Link to="/patterns/filters">filter</Link> functionality. This is especially helpful on small screens like mobile device where space is limited and can’t fit filter components inline with the content.</p>
+            </div>
+        </div>
     </>
 );
 
-const RequireAction = () => (
+const ContentGuidelines = () => (
     <>
-        <h2 id="require-action">Require action</h2>
-        <p>
-            To disable the listener responsible for closing the sheet when a user clicks outside, add <CodeTags type="secondary" code="data-require-action" /> to your Sheet.
-        </p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <SheetComponent id="demo-sheet-1" requireAction>
-                <h2>Sheet component</h2>
-                <p>
-                    This sheet can not be closed by clicking outside its white area.
-                </p>
-            </SheetComponent>{"\n"}
-            <button className="btn btn-primary" type="button" data-sheet-open="demo-sheet-1">Open sheet</button>
-        </ComponentPreview>
+        <h2>Content guidelines</h2>
+        <div className="row">
+            <div className="col-12">
+                <div className="d-flex justify-content-center mt-4 mb-4">
+                    <img src={`${basename}img/documentation/sheet/content-guidelines.png`} alt="Sheet component with sticky header and footer" className="img-fluid"/>
+                </div>
+            </div>
+            <ul>
+                <li>The <b>header</b> should have a title and a close button.</li>
+                <li>The <b>main content</b> area can contain a number of different components and this area can be vertically scrollable.</li>
+                <li>The <b>footer</b> is optional and designed to contain buttons that will lead to certain actions. The footer is also sticky.</li>
+            </ul>
+        </div>
     </>
 );
 
 const JavascriptMethods = () => (
```

Comment:
Maybe adding an extra explanation text here, just like we did with accordion, since this also includes an open and close method?

Reply:
Added some more information under the DeveloperDocumentation section.